var/spack/repos/builtin/packages/py-varifier/package.py

```diff
@@ -0,0 +1,45 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class PyVarifier(PythonPackage):
+    """Variant call verification"""
+
+    homepage = "https://github.com/iqbal-lab-org/varifier"
+    url = "https://github.com/iqbal-lab-org/varifier/archive/refs/tags/v0.3.1.tar.gz"
+
+    version(
+        "master",
+        git="https://github.com/iqbal-lab-org/varifier",
+        branch="master",
+        numeric_equivalent=">",
```

Comment:
What is this argument? Why is that needed?

Reply:
good question... They are not needed.